src/main/java/CamWithLensTest.java

```diff
@@ -0,0 +1,24 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.*;
+
+class CamWithLensTest {
+
+    @Test
+    void testAddToArraylist(){
```

Comment:
der test macht jetzt nicht soviel sinn, denn was du hier testest, ist dass ein array wenn du ein element hinzufügst danach die länge 1 hat - d.h. du testest nur Logik der ArrayList klasse, in der hast du aber überhaupt nichts implementiert. 

Ein sinnvollerer Test wäre, dass du eine Kamera erstellst, danach die verschiedenen variablen prüfst, dann vlt das Objektiv wechselst und wieder die Variablen prüfst.